src/com/mitrais/atm/model/Transaction.java

```diff
@@ -0,0 +1,52 @@
+package com.mitrais.atm.model;
+
+import java.util.Date;
+
+public class Transaction {
+    private Date transactionDate;
```

Comment:
Use java 8+ DateTime API. Date is from previous releases. This was requested in the review page of this stage

Reply:
I have change it into DateTimeAPI 
1fe1f54